packages/helm-charts/prometheus-stackdriver/values.yaml

```diff
@@ -136,6 +136,7 @@ remote_write:
               |coredns.*\
               |cortex.*\
               |csi_operations.*\
+              |eksportisto_step_duration_bucket\
```

Comment:
@pedro-clabs or @martinvol - do you need this in grafana cloud?

Reply:
We are using `eksportisto_step_duration_sum` for sure. If it's possible to filter the bucket part of a histogram metric and leave the sum, I'm not opposed.